elastic/kibana
# [Reporting] upgrade puppeteer to v22.13.1

## Summary

Update for puppeteer, the following changeset updates puppeteer to version `22.13.1`. 

The chromium version required for this version of puppeteer is `126.0.6478.182` from revision `1300313`, as such the chromium binary included for windows and darwin platforms either match or were the closest revision to the expectation. The linux headless binary was built from commit `5b5d8292ddf182f8b2096fa665b473b6317906d5` of the same revision.




### How to verify linux headless build
- clone the following repo https://github.com/tsullivan/kibana-dev-docker
- pull this particular PR
- follow the steps outlined in the repo, replacing any occurrence of `kibana-8.13.0-SNAPSHOT-linux-aarch64.tar.gz` from the repo above's step with the output of running build on this changeset.
- before running step 4, modify the `kibana.yml` file from the `kibana-dev-docker` repo and include the following so we might be able to verify the version of chromium running; 
   ```yaml
     logging.loggers:
       - name: plugins.reporting
         level: debug
   ```
- complete the steps outlined in the README, you'll have a linux distro of kibana running on port `5601`
- Attempt creating exports of PDF and PNG reports, in dashboard, canvas, and visualizations, on report creation attempt we would see a log output that prints out the chromium version exactly matching this;

<img width="1645" alt="Screenshot 2024-07-30 at 16 07 10" src="https://github.com/user-attachments/assets/930a05fa-385b-4429-89e8-7b846043be59">
